src/shared/ResourceForm/fields/TextArrayInput.js

```diff
@@ -34,28 +36,28 @@ export function TextArrayInput({
           internalValue,
           setMultiValue,
         }) => (
-          <FormInput
+          <Input
             placeholder={Math.abs(index) === 1 ? placeholder : ''}
             key={index}
-            compact
             value={value || ''}
             ref={ref}
-            onChange={e => {
+            onInput={e => {
               setValue(e.target.value);
               updateValue();
             }}
-            onKeyDown={e => focus(e)}
-            onBlur={() => {
-              const fieldValue = internalValue?.filter(val => !!val);
-              setMultiValue(
-                typeof customFormatFn === 'function'
-                  ? customFormatFn(fieldValue)
-                  : fieldValue,
-              );
-            }}
-            readOnly={readOnly}
+            className="input-full"
+            // onKeyDown={e => focus(e)}
+            // onBlur={() => {
+            //   const fieldValue = internalValue?.filter(val => !!val);
+            //   setMultiValue(
+            //     typeof customFormatFn === 'function'
+            //       ? customFormatFn(fieldValue)
+            //       : fieldValue,
+            //   );
+            // }}
```

Comment:
this is not adjusted as I see it properly, we need custom functionality for this

Reply:
We might have to adjust it later once this functionality is added to UI5 webcomonents. The workaround is very heavy on memory and the page gets into infinite loop if we try to add eventListener in array